PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -46,15 +46,53 @@ static bool isAllDigits(const generic_string &str)
 	return std::all_of(str.begin(), str.end(), ::isdigit);

```

Comment:
@dail8859 as per documentation ```isdigit``` behavior is undefined if the value of ch is not representable as ```unsigned char``` and is not equal to ```EOF```.

So in case of Asian languages this statement throws assert failure (Though it is irrelevant for release version).
Is there any issue if this is replaced with
``` C++
return str.find_first_not_of("0123456789") == std::string::npos;
```